assets/js/modules/search-console/datastore/index.js

```diff
@@ -25,8 +25,15 @@ import baseModuleStore from './base';
 import report from './report';
 import service from './service';
 import properties from './properties';
+import { areSettingsEditDependenciesLoaded } from './settings';
 
-const store = combineStores( baseModuleStore, report, service, properties );
+const store = combineStores(
+	baseModuleStore,
+	report,
+	service,
+	{ selectors: { areSettingsEditDependenciesLoaded } },
```

Comment:
I know I suggested doing this, but let's export a store partial from `settings` to combine here like the others instead. I should have thought to do this before.